imap_processing/tests/conftest.py

```diff
@@ -102,6 +139,7 @@ def pytest_collection_modifyitems(items):
     +=====================+============================+
     | external_kernel     | _download_external_kernels |
     | use_test_metakernel | use_test_metakernel        |
+    | download_test_data  | _download_test_data        |
```

Comment:
The mark usually tells you what is going on like "requires X", "external data", then the function would do what you need to meet that requirement/expectation. For consistency, I'd suggest going with `external_test_data` indicating this test is marked as needing external test data.
```suggestion
    | external_test_data  | _download_test_data        |
```